src/style.css

```
@@ -1 +1,73 @@
 /* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap");
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  height: 60px;
+}
+
+.nav__ul {
+  display: flex;
+  align-items: center;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__li {
+  margin: 0 10px;
+}
+
+.nav {
+  margin-right: 40px;
+}
+
+.nav__a {
+  color: black;
+  font-size: 12px;
+  text-decoration: none;
+  text-transform: uppercase;
+  padding: 22.5px 0;
+  display: inline-block;
+}
+
+.nav .is-active {
+  color: #00acdc;
+}
+
+.nav__a:hover {
+  color: #00acdc;
+}
+
+.nav__li:first-child {
+  position: relative;
+}
+
+.is-active::after {
+  position: absolute;
+  content: "";
+  display: block;
+  background-color: #00acdc;
+  height: 2px;
+  width: 100%;
+  bottom: 0;
+  border-radius: 20%;
+}
+
+.header__logo {
+  margin-left: 50px;
+}
```

```suggestion
.header__logo {
  display: flex;
  margin-left: 50px;
}
```

You can set `display: flex;` or `height:40px;` to make link size the same as the logo and then aline items.

---

Sorry, I resubmitted the review request and didn't remove those.